WebHost.py

```diff
@@ -63,7 +121,9 @@ def create_ordered_tutorials_file() -> typing.List[typing.Dict[str, typing.Any]]
         logging.warning("Could not update LttP sprites.")
     app = get_app()
     create_options_files()
+    # TODO create a proper hook for world documents so they don't need to be copied
```

Comment:
Technically the more efficient way is to copy them into static for nginx to pick up the work from there.

Reply:
No, I'm saying to get rid of the TODO entry.